backend/src/test/repository/BaseRepositoryTest.ts

```diff
@@ -1,9 +1,9 @@
-import { Connection, createConnection } from 'typeorm';
+import { createConnection, EntityMetadata, getConnection } from 'typeorm';
 
 // use this class whenever we want to write unit/integration tests that interact with foodies_test database.
-export class BaseRepositoryTest {
-  static establishTestConnection(): Promise<Connection> {
-    return createConnection({
+const connection = {
+  async create(): Promise<void> {
+    await createConnection({
```

Comment:
This is just a connection object with three methods
```
create, close, clear
```

`close` calls `clear` beforehand